tests/cli/init-server.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+CLIENT_EXE=basecli
+SERVER_EXE=basecoin
+
+test01initOption() {
```

Comment:
nice test that it writes the values. however, it would be good to make sure it started up and could parse this file.
i guess at least using `jq` to check the genesis file ensures it is proper json, but still, maybe just start the basecoin server and make sure it doesn't return an error.

Reply:
added